app/code/Magento/UpwardConnector/etc/frontend/di.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <preference for="Magento\Framework\App\FrontControllerInterface" type="Magento\UpwardConnector\Controller\Upward" />
+    <type name="Magento\Upward\Controller">
+        <arguments>
+            <argument name="upwardConfig" xsi:type="string">pwa/upward-config.yml</argument>
```

Comment:
Improvement option: I didn't see an easy way to source an argument from ENV, but we can write a simple class that does this, so this value isn't hard-coded.